Spika/Common/Views/MessageInput/InputTextAndControlsView.swift

```diff
@@ -60,17 +60,17 @@ extension InputTextAndControlsView: BaseView {
     }
     
     func positionSubviews() {
-        alignment = .bottom
+        alignment = .center
         spacing = 10
-        directionalLayoutMargins = NSDirectionalEdgeInsets(top: 12, leading: 11, bottom: 12, trailing: 11)
+        directionalLayoutMargins = NSDirectionalEdgeInsets(top: 12, leading: 10, bottom: 12, trailing: 10)
         isLayoutMarginsRelativeArrangement = true
         
```

Comment:
Magic number like this should be avoided.

Reply:
needs to be refactored throughout the whole app